descwl_coadd/coadd.py

```diff
@@ -382,37 +419,43 @@ def _make_coadds(self):
         """
         make warps and coadds for images and noise fields
         """
+        image_data = self._make_warps(
+            exps=self.exps,
+            dims=self.coadd_dims,
+            wcs=self.coadd_wcs,
+            dopsf=True,
+        )
+
         psf_data = self._make_warps(
             exps=self.psf_exps,
             dims=self.psf_dims,
             wcs=self.coadd_psf_wcs,
             dopsf=False,
         )
-        coadd_psf_exp = self._make_coadd(**psf_data)
 
-        pimage = coadd_psf_exp.image.array
-        wbad = np.where(~np.isfinite(pimage))
+        noise_data = self._make_warps(
+            exps=self.noise_exps,
+            dims=self.coadd_dims,
+            wcs=self.coadd_wcs,
+            dopsf=True,
+        )
+
+        # we need the weights in the coadds to be the same
+        # so we replace them here
+        psf_data["weights"] = image_data["weights"]
```

Comment:
I think we need to make it more clear in the commit  messages that this bug fix was put in

Reply:
I'll squash the history on merge and make a nice message

Reply:
We should probs start keeping a changelog too.